packages/tools/src/tools/segmentation/strategies/fillCircle.ts

```diff
@@ -88,9 +61,9 @@ function fillCircle(
       }
 
       if (
-        isWithinThreshold(index, imageVolume, strategySpecificConfiguration)
+        isWithinThreshold(index, imageScalarData, strategySpecificConfiguration)
```

Comment:
This code would all be shared if we merged the dynamic brush strategies first.  Would make it a bit easier to manage.